packages/client/src/Components/PaperEdits/PaperEdit/ProgramScript/index.js

```diff
@@ -24,7 +25,7 @@ class ProgramScript extends Component {
         <h2>Program Script</h2>
         <hr/>
         <h3>Paper Edit title</h3>
-        <section style={ { width: '100%', height:'10em', backgroundColor: 'black', overflow: 'scroll' } } ></section>
+        <PaperCutsPlayer width={  } />
```

Comment:
This throws an error + playlist is missing?

Got it working with this:
``` 
 const playlist = [
      { type:'video', offset:0, start:0, duration:10, src:'http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ElephantsDream.mp4' },
      { type:'video', offset:0, start:5, duration:10, src:'http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4' },
      { type:'video', offset:10, start:10, duration:10, src:'http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ForBiggerBlazes.mp4' }
    ];
...
<PaperCutsPlayer
   playlist={ playlist }
   // width={ 300 }
   />
```

Reply:
oh heck yeah I'm not with it today

Reply:
Do you want it pulling in as part of this PR?